nucypher/cli/characters/alice.py

```diff
@@ -129,14 +129,24 @@ def alice(click_config,
 
     # Switch to character control emitter
     if click_config.json_ipc:
-        ALICE.controller.emitter = IPCStdoutEmitter(quiet=click_config.quiet)
+        ALICE.controller.emitter = JSONRPCStdoutEmitter(quiet=click_config.quiet)
 
     if action == "run":
-        """Start Alice Web Controller"""
-        ALICE.controller.emitter(message=f"Alice Verifying Key {bytes(ALICE.stamp).hex()}", color="green", bold=True)
-        controller = ALICE.make_web_controller(crash_on_error=click_config.debug)
-        ALICE.log.info('Starting HTTP Character Web Controller')
-        return controller.start(http_port=http_port, dry_run=dry_run)
+        """Start Alice Controller"""
+
+        # RPC
+        if click_config.json_ipc:
+            rpc_controller = ALICE.make_rpc_controller()
+            _transport = rpc_controller.make_control_transport()
+            rpc_controller.start()
+            return
+
+        # HTTP
+        else:
+            ALICE.controller.emitter(message=f"Alice Verifying Key {bytes(ALICE.stamp).hex()}", color="green", bold=True)
+            controller = ALICE.make_web_controller(crash_on_error=click_config.debug)
```

Comment:
This seems like we are further overloading the "debug" argument.  Maybe that's fine?

Reply:
Yeah - Let's ensure `--debug` does not become "overloaded".  At the moment It:

- shows logs in the console
- allows otherwise swallowed exceptions to be raised and crash the character control server

Reply:
See related #941